src/v0/sources/shopify/config.js

```diff
@@ -24,7 +24,7 @@ const RUDDER_ECOM_MAP = {
 
 const SHOPIFY_TRACK_MAP = {
   checkouts_delete: 'Checkout Deleted',
-  carts_create: 'Cart Create',
+  // carts_create: 'Cart Create',
```

Comment:
Why is this commented? Is there a later plan to re-introduce it?

Reply:
Nope. 
Removed